packages/core/src/Cardano/types/Block.ts

```diff
@@ -1,9 +1,26 @@
-import { BlockNo, BlockSize, Slot } from '@cardano-ogmios/schema';
-import { Epoch, Lovelace, PoolId } from '.';
 import { Hash28ByteBase16, Hash32ByteBase16, OpaqueString, typedBech32 } from '../util';
 import { InvalidStringError } from '../..';
+import { Lovelace, PoolId } from '.';
 
-export { BlockNo } from '@cardano-ogmios/schema';
+/**
+ * The block size in bytes
+ */
+export declare type BlockSize = number;
```

Comment:
I think `declare` is not needed here (and in other changes in this PR). This keyword is used to tell tsc that some variable is available as constant during runtime, typically used for global variables

```suggestion
export type BlockSize = number;
```

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/388/commits/34fce0e418d63211ce69197f26e5ce198c2f5600